sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -271,7 +271,7 @@ case class AlterTableUnsetPropertiesCommand(
       }
     }
```

Comment:
In above code, we check if a prop key exists in table.properties or not when `ifExists` is false. As `comment` is not in table.properties, I think it will cause `AnalysisException` when unset comment?